yt_dlp/extractor/litv.py

```diff
@@ -5,59 +5,51 @@
     ExtractorError,
     int_or_none,
     smuggle_url,
-    traverse_obj,
     unsmuggle_url,
 )
 
 
 class LiTVIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?litv\.tv/(?:vod|promo)/[^/]+/(?:content\.do)?\?.*?\b(?:content_)?id=(?P<id>[^&]+)'
 
-    _URL_TEMPLATE = 'https://www.litv.tv/vod/%s/content.do?id=%s'
+    _URL_TEMPLATE = 'https://www.litv.tv/vod/%s/content.do?content_id=%s'
 
     _TESTS = [{
         'url': 'https://www.litv.tv/vod/drama/content.do?brc_id=root&id=VOD00041610&isUHEnabled=true&autoPlay=1',
         'info_dict': {
             'id': 'VOD00041606',
             'title': '花千骨',
         },
-        'playlist_count': 50,
+        'playlist_count': 51,  # 50 episodes + 1 trailer
     }, {
         'url': 'https://www.litv.tv/vod/drama/content.do?brc_id=root&id=VOD00041610&isUHEnabled=true&autoPlay=1',
-        'md5': '969e343d9244778cb29acec608e53640',
+        'md5': 'b90ff1e9f1d8f5cfcd0a44c3e2b34c7a',
         'info_dict': {
             'id': 'VOD00041610',
             'ext': 'mp4',
             'title': '花千骨第1集',
             'thumbnail': r're:https?://.*\.jpg$',
-            'description': 'md5:c7017aa144c87467c4fb2909c4b05d6f',
+            'description': '《花千骨》陸劇線上看。十六年前，平靜的村莊內，一名女嬰隨異相出生，途徑此地的蜀山掌門清虛道長算出此女命運非同一般，她體內散發的異香易招惹妖魔。一念慈悲下，他在村莊周邊設下結界阻擋妖魔入侵，讓其年滿十六後去蜀山，並賜名花千骨。',
+            'categories': ['奇幻', '愛情', '中國', '仙俠'],
+            'episode': 'Episode 1',
             'episode_number': 1,
         },
         'params': {
             'noplaylist': True,
         },
         'skip': 'Georestricted to Taiwan',
-    }, {
-        'url': 'https://www.litv.tv/promo/miyuezhuan/?content_id=VOD00044841&',
-        'md5': '88322ea132f848d6e3e18b32a832b918',
-        'info_dict': {
-            'id': 'VOD00044841',
-            'ext': 'mp4',
-            'title': '芈月傳第1集　霸星芈月降世楚國',
-            'description': '楚威王二年，太史令唐昧夜觀星象，發現霸星即將現世。王后得知霸星的預言後，想盡辦法不讓孩子順利出生，幸得莒姬相護化解危機。沒想到眾人期待下出生的霸星卻是位公主，楚威王對此失望至極。楚王后命人將女嬰丟棄河中，居然奇蹟似的被少司命像攔下，楚威王認為此女非同凡響，為她取名芈月。',
-        },
-        'skip': 'Georestricted to Taiwan',
```

Comment:
why was this removed? it was already being skipped

Reply:
I removed that particular test because the video it tries to download does not exist any more, so it will fail regardless of georestriction. I left tests that are (currently) passing if you use yt-dlp from Taiwan or using a proxy. In other words, you can still run the tests of this extractor after commenting out the `'skip': 'Georestricted to Taiwan',` lines.

Reply:
Ah I see. Can you add the test back, but change the `skip` value from "Georestricted to Tawain" to "No longer exists"